RequestKit.xcodeproj/project.pbxproj

```diff
@@ -867,6 +868,7 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				SKIP_INSTALL = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
+				SWIFT_VERSION = 3.0;
```

Comment:
thanks for all the info! so, the entry needs to be set in the project build settings but not in the targets?! which value should it have in the project build settings then? my latest commit contains the change where I removed it from all the target build settings, but I haven't readded it to the project build settings as I'm not quite sure what value to set... sorry for the confusion and thanks for taking the time to provide instructions 😔